src/types/any.rs

```diff
@@ -1855,6 +1907,38 @@ impl<'py> PyAnyMethods<'py> for Bound<'py, PyAny> {
             .and_then(|any| any.is_truthy())
     }
 
+    implement_binop!(add, PyNumber_Add, "+");
+    implement_binop!(sub, PyNumber_Subtract, "-");
+    implement_binop!(mul, PyNumber_Multiply, "*");
+    implement_binop!(div, PyNumber_TrueDivide, "/");
+    implement_binop!(bitand, PyNumber_And, "&");
```

Comment:
We seem to be missing bitor, xor, modulo, matrix multiplication etc. Was there a reason you limited to just these ones rather than the full list?

Reply:
There was a reason (unclear if it was a _good_ reason): I did the basic arithmetic ops + the set of things I needed :-)

Reply:
Heh, fair enough. Maybe let's at least in this PR list them all down and agree what makes sense to include to have a consistent set if we don't do everything? For example, I think at least bitor makes sense if we have bitand 😂